pkgs/development/python-modules/langchain-core/default.nix

```diff
@@ -76,6 +69,12 @@ buildPythonPackage rec {
 
   pytestFlagsArray = [ "tests/unit_tests" ];
 
+  # don't add langchain-standard-tests to nativeCheckInputs
+  # to avoid circular import
+  preCheck = ''
+    export PYTHONPATH=${src}/libs/standard-tests:$PYTHONPATH
+  '';
+
```

Comment:
Nice trick ! I did that in the PR I merged yesterday because I wanted to make a package out of `langchain-standard-tests` but I failed and left it as it was. This is a cool improvement!